documentation/book/oauth/proc-oauth-broker-config.adoc

```diff
@@ -0,0 +1,108 @@
+// Module included in the following assemblies:
+//
+// assembly-oauth-strimzi-config.adoc
+
+[id='proc-oauth-broker-config-{context}']
+= Configuring {oauth} support for Kafka brokers
+
+This procedure describes how to configure Kafka brokers so that the broker listeners are enabled to use {oauth} authentication using an authorization server.
+
+In this scenario, {ProductName} and the authorization server are running in the same cluster.
+
+If the authorization server is using self-signed certificates with valid hostnames, TLS connection works out of box.
+Otherwise, you have two connection options for your external listener configuration:
+
+. TLS connection to the authorization server with trusted certificates
+. Direct connection using an introspection endpoint configuration
+
+Both options are described in this procedure.
+
+
+.Before you start
+
+For more information on the configuration and authentication of Kafka broker listeners, see:
+
+* xref:assembly-configuring-kafka-broker-listeners-deployment-configuration-kafka[Kafka broker listeners]
+* xref:assembly-kafka-authentication-and-authorization-deployment-configuration-kafka[Authentication and Authorization]
+
+.Prerequisites
+
+* {ProductName} and Kafka is running
+* {oauth} authorization server is deployed
+
+.Procedure
+
+. Update the Kafka broker configuration (`Kafka.spec.kafka`) of your `kafka` resource in an editor.
++
+[source,shell]
+----
+kubectl edit kafka my-cluster
+----
+
+. Configure the Kafka broker listener configuration
```

Comment:
Could you please clarify that the configuration is independent for each listener? It doesn't have to be the same.